Lesson_204V2/StartSolution/MainPage.xaml.cs

```diff
@@ -0,0 +1,110 @@
+﻿using System;
+using System.Diagnostics;
+using System.Net.Http;
+using System.Threading;
+using System.Threading.Tasks;
+using Windows.Devices.Adc;
+using Windows.Media.SpeechSynthesis;
+using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Navigation;
+
+using Microsoft.IoT.AdcMcp3008;
+// The Blank Page item template is documented at http://go.microsoft.com/fwlink/?LinkId=402352&clcid=0x409
```

Comment:
comment no longer needed here as well, as noticed by @mwmckee in FullSolution (this is StartSolution)